fbsat-core/src/main/kotlin/ru/ifmo/fbsat/core/task/single/complete/Cegis.kt

```diff
@@ -12,9 +12,7 @@ import ru.ifmo.fbsat.core.task.single.extended.ExtendedTask
 import ru.ifmo.fbsat.core.task.single.extended.extendedMin
 import ru.ifmo.fbsat.core.task.single.extended.extendedMinUB
 import ru.ifmo.fbsat.core.task.single.extended.inferExtended
-import ru.ifmo.fbsat.core.utils.Globals
-import ru.ifmo.fbsat.core.utils.MyLogger
-import ru.ifmo.fbsat.core.utils.timeSince
+import ru.ifmo.fbsat.core.utils.*
```

Comment:
Please, setup your IDEA not to fold imports under star-import. Generally, it is better to specify all imports **explicitly** -- anyway, the IDE does this for you. On the other hand, I already forbid star-imports in my settings, so my next commit will include those changes anyway due to auto-formatting on my side.